purchases/src/main/kotlin/com/revenuecat/purchases/BillingWrapper.kt

```diff
@@ -253,9 +253,13 @@ internal class BillingWrapper internal constructor(
         return billingClient?.let {
             debugLog("[QueryPurchases] Querying $skuType")
             val result = it.queryPurchases(skuType)
+
+            // Purchases.PurchaseResult#purchasesList is not marked as nullable, but it does sometimes return null.
+            val purchasesList = if (result.purchasesList != null) result.purchasesList else emptyList<Purchase>()
```

Comment:
Can you use the Elvis operator here?

Reply:
Done! (Also shows how much Kotlin I write 🤦‍♂ )